### ISSUE TITLE
Supernatant extraction: When should I recombine samples into 1? 

### ISSUE THREAD

**Issue by grace-ac** [RobertsLab/resources#581]
Haven't gotten to isolating RNA part yet, but when I do, 

### at what point should I recombine the four tubes into one per sample?? 

Should I do it after resuspension or after incubation? Does it matter? 

![screen shot 2019-02-14 at 3 41 39 pm](https://user-images.githubusercontent.com/14934314/52824878-51991200-306f-11e9-8dde-db7b67018d7a.png)

**Comment by kubu4** [comment]
What protocol are you using for RNA isolation?

**Comment by kubu4** [comment]
I'm guessing this is Trizol LS. If yes, then sample recombination should happen at the point of precipitation. Like so:

1. Crab sample 503
2. Split in to four different preps (250uL each).
- 503-1
- 503-2
- 503-3
- 503-4

3. Perform lysis and separation steps on each of the four preps.
4. Begin precipitation:
- add isopropanol to each tube and mix thoroughly
- incubate all tubes for at least 10mins
5. Pellet RNA in _one_ of the tubes (e.g. 503-1)
6. Remove supernatant from pelleted RNA.
7. Transfer contents of next tube to first tube (e.g. transfer contents of 503-2 to 503-1 tube)
8. Pellet RNA.
9. Repeat the transferring steps until all samples have been pelleted in tube 503-1.
10. Continue remainder of procedure.

Post here if you have questions.

**Comment by grace-ac** [comment]
@kubu4 Ah yes- sorry!! Trizol LS with supernatant samples. 

Should have assigned @sr320 and/or @shellytrigg because we were talking about it today, so I didn't get into too many details. 

Your comment makes sense! Will make rest of protocol easier to keep track of because I won't be dealing with 16 tubes anymore

**Comment by sr320** [comment]
Double check with @shellytrigg

Thanks
steven

**Comment by shellytrigg** [comment]
@grace-ac and I just talked about it and to prevent tubes from sitting or spinning too long, we talked about keeping all tubes separate and processing simultaneously until the very end after RNA pellets are air dried. To recombine the 4 tubes from 1 animal, resuspend (pipetting up and down) 1 pellet in 20-50uL of RNA RNase-free water or 0.1 mM EDTA while leaving the other pellets drying. Then use that same 20-50uL of resuspended pellet to resuspend the next RNA pellet. And continue to resuspend the rest of the dried pellets in the same 20-50uL. You end up with only 20-50uL final volume containing 4 pellets.
